src/app/members/details/[id]/page.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { MemberDetails } from "../../components/MemberDetails";
+import { getUserById } from "../../actions/get-user-by-id";
+import FormWrapper from "@/common/FormWrapper";
+
+type Parameters = {
+  params: {
+    id: string;
+  };
+};
+
+const Page = async ({ params }: Parameters) => {
+  const { id } = params;
+  const userInfo = await getUserById(id);
+  return (
+    <div className="p-8">
+     <FormWrapper><MemberDetails userInfo={userInfo} /></FormWrapper>  
```

Comment:
Why we need form wrapper here ?

Reply:
Because this detail page has a similar layout as the form.